docs/docs/installation/running-on-kubernetes.mdx

```diff
@@ -89,7 +89,10 @@ Default security settings and passwords are included but you **SHOULD** override
 
 ```yaml
 postgresql:
-  postgresqlPassword: superset
+  auth:
+    postgresPassword: superset
+    password: superset
```

Comment:
In my opinion, it should be made clear that this is only one way to pass credentials to the Bitnami Postgres chart that the Superset chart depends on. In our deployment, we have an external process (Terraform) that creates Secrets for us, so we use the `existingSecret` mechanism from the Postgres chart, which imo is more secure since we're not committing any plaintext password to Git. I think that mentioning a few of the options and linking to the [Postgres chart's documentation](https://artifacthub.io/packages/helm/bitnami/postgresql) or even the [Postgres chart's values file](https://github.com/bitnami/charts/blob/main/bitnami/postgresql/values.yaml) would be much more suitable. It is also worth noting that the Postgres chart is an optional dependency and people are free to "bring their own" Postgres...